packages/turf-boolean-is-valid/bench.js

```diff
@@ -0,0 +1,29 @@
+import path from 'path';
```

Comment:
For now, we would need to convert the `bench.js` & `test.js` into CommonJS.

We "could" convert them to Typescript, however lets just focus on the `index.js` file for now and we can worry about the Bench & Test files later.